modules/socials/consts/Facebook.ts

```diff
@@ -0,0 +1,20 @@
+type Facebook = {
```

Comment:
Hi @RizkyRajitha can you add OpenApi Comments to this model and reference it in the api endpoint please :P

Reply:
Please can you also add unit tests for the code you have submitted. 👍 Thank you.

Great contribution.